src/header/js/header-menus.js

```diff
@@ -265,19 +307,31 @@ export default ({variant = 'desktop'}) => {
 						))
 				);
 			}
-			$items.push(
-				$('li', {class: `${prefix}-subitem`},
-					$('a', {href: entry.href, class: `${prefix}-sublink`},
-						$('p', {class: `${prefix}-sublink--title`},
-							entry.label
-						),
-						$('p', {class: `${prefix}-sublink--description`},
-							entry.description
+			
+			if (entry.href && entry.label && entry.description) {
+				$items.push(
+					$items.push(
+						$('li', {class: `${prefix}-entry--menus-subitem`},
+							$('a', {href: entry.href, class: `${prefix}-entry-sublink`},
+								$('p', {class: `${prefix}-entry-sublink--title`}, entry.label),
+								$('p', {class: `${prefix}-sublink--description`}, entry.description)
```

Comment:
You could do something like the following instead and remove the check for `entry.description` in the if statement above.
```js
entry.description ? $('p', {class: `${prefix}-sublink--description`}, entry.description) : ""
```